### DISCUSSION TITLE
possibility to use key events on MultiSelect control

### DISCUSSION THREAD

**Discussion by anna777x** [dotnetcore/BootstrapBlazor#3609] [Q&A]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Is your feature request related to a problem? Please describe the problem.

I have edit form. And user wants to go though controls by clicking 'tab' on keyboard. so when user is clicking on MultiSelect control, last one should open.
Another situation: user's selecting one option in MultiSelect control and clicking 'Enter' on keyboard, after that form should submit.

### Describe the solution you'd like

It would be nice if MultiSelect /Select control have method 'OnTabAsync' and 'OnEnterAsync'

as you see I'm actively using these components from your fantastic Blazor Zone library. 
Thank you so much for this incredible library.

**Comment by bb-auto** [comment]
@anna777x Thank you for contacting us. We will give feedback later.

**Comment by ArgoZhang** [comment]
@anna777x could you give a star for this project? thanks

**Comment by ArgoZhang** [comment]
@anna777x hi. thank you for contacting us. let us make your requirements clear.

1. I guess you mean that auto expands the dropdown when the user tabs <kbd>**Tab**</kbd> keyboard the `MultiSelect` component has focus.
2. in the `EditorForm` component the user presses `Enter` to submit the form

is right?

**Comment by anna777x** [comment]
you understand me absolutely correct!
